databases/week3/additional_meal_queries.sql

```diff
@@ -0,0 +1,52 @@
+USE hyf_lesson3;
+
+SELECT *
+FROM  meal 
+WHERE  price  < 40;
+SELECT *

```

Comment:
Having available reservations basically means that the sum of all guests for a given meal is less than max_reservations.

This is the trap query btw. You need to sum the number_of_guests over all reservations for a given meal and then compare that number with the max_reservations of that meal.

There are at least two ways of doing this, both involve stuff that we didnt talk about in class:
 - use the COALESCE() function: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce
 - use GROUP BY, SUM, and `HAVING: https://www.w3schools.com/sql/sql_having.asp

Also remember that there might be meals that don't have any reservations yet.